site/src/sphinx/server-annotated-service.rst

```diff
@@ -434,6 +438,54 @@ converting ``MyRequest``.
             @RequestObject(MySpecialRequestConverter.class) MyRequest myRequest) { ... }
     }
 
+
+Injecting value of parameters and HTTP headers into a Java object
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+BeanRequestConverterFunction_ is a built-in request converter for Java object. You can use it by putting
+`@RequestObject`_ annotation on the parameters which you want to convert.
+
+.. code-block:: java
+
+    public class MyAnnotatedServiceClass {
+        @Post("/hello")
+        public HttpResponse hello(@RequestObject MyRequestObject myRequestObject) { ... }
+    }
+
+Beside this, you also need to inject value of path parameters, HTTP parameters, or HTTP headers to the Java
```

Comment:
you can also inject values of ?

Reply:
I think he is trying to tell the reader that the reader should also add some annotations to `MyRequestObject`. How about rephrasing this way:

```
Besides the annotated service class, you also need to create ``MyRequestObject`` and add the @Param and @Header annotations to it to inject the path parameters, HTTP parameters or HTTP headers
```

Reply:
Yes, that's what I am trying to say.
I will fix it tomorrow.

Reply:
fixed